src/Annotation/AnnotationPanel.jsx

```diff
@@ -35,6 +35,8 @@ export default function AnnotationPanel(props) {
     const index = parseInt(newValue, 10);
     if (config.layers[index]) {
       actions.selectViewerLayer(config.layers[index].name);
+    } else {
```

Comment:
Why call the else statement. Does something bad happen if we call nothing at all when the if condition is not met?

Reply:
I added it on purpose to disable annotation layer selection when switching to merge/connection tab. If the annotation layer were kept on, the user might be confused by seeing an annotation added while [ctrl]clicking on the view accidentally.